src/gui/properties/trackerlist.cpp

```diff
@@ -284,8 +287,18 @@ void TrackerList::loadTrackers() {
         item->setText(COL_MSG, "");
         break;
     }
+    item->setText(COL_RECEIVED, QString::number(data.numPeers));
+
+#if LIBTORRENT_VERSION_NUM >= 10000
```

Comment:
@ngosang Replace `>= 10000` with `>= 10100` here.  The scrape problem is fixed in Libtorrent v1.1 (https://github.com/arvidn/libtorrent/pull/433).  

Thanks, 
~ Nyriox